RocketChat/Rocket.Chat
# [NEW] Permissions for interacting with Omnichannel Contact Center

## Proposed changes (including videos or screenshots)


Adds a new permission, one that allows for control over user access to Omnichannel Contact Center,



## Issue(s)


## Steps to test or reproduce

The user with the view-omnichannel-contact-center permission should be allowed to open the contact center, those without such permission, should not.

## Further comments
